tool_collections/samtools/cram_to_bam/samtools_cram_to_bam.xml

```diff
@@ -0,0 +1,116 @@
+<tool id="samtools_cram_to_bam" name="samtools CRAM to BAM" version="1.2.0">
+    <description>convert CRAM alignments to BAM format</description>
+
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements"></expand>
+    <expand macro="version_command"></expand>
+    <expand macro="stdio"></expand>
+
+    <command>
+    <![CDATA[
+        #if str( $input_alignment.metadata.cram_index ) != "None":
+            ln
+                -f
+                -s
+                "${input_alignment.metadata.cram_index}"
+                "${input_alignment}.crai"
+                &&
+        #end if
+        
+        samtools view
+            #if $parameter_regions.target_region == "regions_bed_file"
+                -L $parameter_regions.regions_bed_file
+            #end if
+            
+            -@ \${GALAXY_SLOTS:-2}
+            -b
+            -T ${reference_source.input_reference}
+            -o $output_alignment
+            $input_alignment
```

Comment:
`${reference_source.input_reference}`, `$output_alignment` and `$input_alignment` should be quoted.